devel/synfig/Makefile

```diff
@@ -23,7 +23,7 @@ LIB_DEPENDS=	libImath.so:math/Imath \
 		libfontconfig.so:x11-fonts/fontconfig \
 		libfreetype.so:print/freetype2 \
 		libltdl.so:devel/libltdl \
-		libmlt.so:multimedia/mlt \
+		libmlt.so:multimedia/mlt6 \
```

Comment:
I suggest creating a Mk/Uses/mlt with version-arg

So that ports use USES=mlt:6 or USES=mlt:7,qt

Reply:
see for example eigen.mk which is pretty similar to that